prowler/providers/azure/services/containerregistry/containerregistry_network_access_restricted/containerregistry_network_access_restricted.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.containerregistry.containerregistry_client import (
+    containerregistry_client,
+)
+
+
+class containerregistry_network_access_restricted(Check):
+    def execute(self) -> list[Check_Report_Azure]:
+        findings = []
+
+        for subscription, registries in containerregistry_client.registries.items():
+            for registry_id, ContainerRegistryInfo in registries.items():
```

Comment:
```suggestion
            for registry_id, container_registry_info in registries.items():
```

Please, use lower case and underscores for variables to follow our code style.